src/api/index.spec.tsx

```diff
@@ -489,4 +506,47 @@ describe(__filename, () => {
       }).toThrow(/Either `path` or `url` must be defined/);
     });
   });
+
+  describe('makeQueryString', () => {
+    it('transforms an object to a query string', () => {
+      const query = makeQueryString({ user: 123, addon: 321 });
+      expect(query).toContain('user=123');
```

Comment:
ooh, I see that we have the fancy expect helper already. This test (and the others, where applicable) should use it:

```diff
diff --git a/src/api/index.spec.tsx b/src/api/index.spec.tsx
index 854fe7d..1d03db5 100644
--- a/src/api/index.spec.tsx
+++ b/src/api/index.spec.tsx
@@ -510,8 +510,9 @@ describe(__filename, () => {
   describe('makeQueryString', () => {
     it('transforms an object to a query string', () => {
       const query = makeQueryString({ user: 123, addon: 321 });
-      expect(query).toContain('user=123');
-      expect(query).toContain('addon=321');
+      expect(query).toEqual(
+        expect.urlWithTheseParams({ user: '123', addon: '321' }),
+      );
     });
 
     it('ignores undefined query string values', () => {
```